src/components/CardContainer/CardContainer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import Card from "../Card/Card"
+import UniqueRandomArray from "unique-random-array"
+
+class CardContainer extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            dogs: []
+        }
+    }
+
+    componentDidMount() {
+        this.getDogs()
+    }
+
+    getDogs = () => {
+        fetch('http://localhost:3000/dogs', {
+            method: 'get'
+        })
+            .then((data) => data.json())
```

Comment:
As there's only parameter for our function, we can lose the brackets around `data` and it'll work the same. Also on lines 23, 30, 43

Reply:
on it